expertise/create_dataset.py

```diff
@@ -26,12 +26,71 @@ def convert_to_list(config_invitations):
     assert isinstance(invitations, list), 'Input should be a str or a list'
     return invitations
 
-def get_publications(openreview_client, author_id):
+def get_publications(openreview_client, config, author_id):
     content = {
         'authorids': author_id
     }
     publications = openreview.tools.iterget_notes(openreview_client, content=content)
-    return [publication for publication in publications]
+
+    # Get all publications and assign tcdate to cdate in case cdate is None. If tcdate is also None
+    # assign cdate := 0
+    unsorted_publications = []
+    for publication in publications:
+        if getattr(publication, 'cdate') is None:
+            publication.cdate = getattr(publication, 'tcdate', 0)
+        unsorted_publications.append(publication)
+
+    # If the author does not have publications, then return early
+    if len(unsorted_publications) == 0:
```

Comment:
Minor comment, let's write this in a more pythonic style.

```suggestion
    if not unsorted_publications:
```